src/libs/tabbable.ts

```diff
@@ -1,10 +1,10 @@
 const candidateSelectors = [
-  'input',
-  'select',
-  'textarea',
+  'input:not([disabled])',
+  'select:not([disabled])',
+  'textarea:not([disabled])',
   'a[href]',
-  'button',
-  '[tabindex]',
+  'button:not([disabled])',
+  '[tabindex="0"]',
```

Comment:
If the `tabindex` is a positive value, the element became focusable.
And if `tabindex="-1"`, the element is not reachable via keyboard navigation, but could be focused with js or clicking. We are using this behavior for [dummy element](https://github.com/kufu/smarthr-ui/blob/master/src/components/Dialog/DialogContentInner.tsx#L124-L125) for focus management in Dialog, the change of this line is causing a [difference](https://smarthr-ui-snapshots.s3.amazonaws.com/1966c64b90889d97504278893500743c7103deab/index.html?stat=q1ZKS8zMSU3xLEnNLXbOL80rUbIy0lHKSy1HFjHQUUpJzUktQVUHFC1ILC5GFbQwqAUA) in the scroll position of the content when the `Dialog` opened.

Therefore, I think the change of this line is not necessary.

```suggestion
  '[tabindex]',
```

Reply:
OK. I fixed the problem.